orchestrator-bundle/orc8r-nginx-operator/tests/integration/test_integration.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import logging
+from pathlib import Path
+
+import pytest
+import yaml
+from pytest_operator.plugin import OpsTest  # type: ignore[import]  # noqa: F401
+
+logger = logging.getLogger(__name__)
+METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
+CONTROLLER_METADATA = yaml.safe_load(
+    Path("../orc8r-controller-operator/metadata.yaml").read_text()
+)
+APPLICATION_NAME = "orc8r-nginx"
+CHARM_NAME = "magma-orc8r-nginx"
```

Comment:
Shouldn't be the `CHARM_NAME` `magma-orc8r-nginx-lite` as it is in `metadata.yaml`. Maybe I'm missing something.

Reply:
Yah I changed the variable names as it was confusing. The image name is `magma-orc8r-nginx-image`.